File: LR_3-4/2.7.pas
program twopointseven;

var i, N : integer;
length, width, area, totalarea, S : real;

begin
  Readln(N);
  totalarea := 0;
  i := 1;
  while i<=N do begin
    writeln(i);
    readln(length);
    writeln(i);
    readln(width);
    area := length * width;
    i := i+1;
    writeln(area);
    S := S+area;
  end;
  writeln('totalarea = ', S);
  
end.

File: LR_3-4/2.7.py

N = int(input())
total_area = 0.0
S = 0.0
i = 0

while i<N:
    length = float(input())
    width = float(input())
    area = round(length * width)
    total_area += area
    i += 1
    print(area)

print('total area =', total_area)